numpy/core/src/common/npy_partition.h.src

```diff
@@ -80,9 +90,11 @@ static part_map _part_map[] = {
         NPY_@TYPE@,
         {
             (PyArray_PartitionFunc *)&introselect_@suff@,
+            (PyArray_PartitionFunc *)&floydselect_@suff@
```

Comment:
This is ABI-incompatible, IIUC.

Reply:
@seberg Won’t this change the slots in an ABI incompatible way?

Reply:
Could you elaborate pls, I don't understand.